services/wallet/bridge/hop.go

```diff
@@ -292,22 +292,36 @@ func (h *HopBridge) swapAndSend(chainID uint64, hopArgs *HopTxArgs, signerFn bin
 		return tx, err
 	}
 
+	toNetwork := h.contractMaker.RPCClient.NetworkManager.Find(hopArgs.ChainID)
+	if toNetwork == nil {
+		return tx, err
+	}
+
 	txOpts := hopArgs.ToTransactOpts(signerFn)
 	if token.IsNative() {
 		txOpts.Value = (*big.Int)(hopArgs.Amount)
 	}
 	now := time.Now()
 	deadline := big.NewInt(now.Unix() + 604800)
+	amountOutMin := big.NewInt(0)
+	destinationDeadline := big.NewInt(now.Unix() + 604800)
+	destinationAmountOutMin := big.NewInt(0)
+
+	if toNetwork.Layer == 1 {
+		destinationDeadline = nil
+		destinationAmountOutMin = nil
```

Comment:
We can safely pass `big.NewInt(0)`

Just checked on Hop explorer a tx Optimism to Mainnet.

This is a Bonded tx from Optimism to Mainnet: 
```
#    Name    Type    Data
0    chainId    uint256    1
1    recipient    address    0xa8bd82A0Bd23206F707407276d08fAF44879BA57
2    amount    uint256    600000000000000000
3    bonderFee    uint256    11515474892198123
4    amountOutMin    uint256    584907222787869573
5    deadline    uint256    1709733092
6    destinationAmountOutMin    uint256    0
7    destinationDeadline    uint256    0
```

https://optimistic.etherscan.io/tx/0xebe3e52df8132ba6fb875b0e550af502f1d7bfd82b8a38b0fbf0e02271147b5c